content/100-getting-started/01-quickstart.mdx

```diff
@@ -102,19 +102,18 @@ At this point, you have a Prisma schema but no database yet. Run the following c
 npx prisma migrate dev --name init
 ```
 
-This command did two things:
+This command did three things:
 
 1. It creates a new SQL migration file for this migration in the `prisma/migrations` directory.
-1. It runs the SQL migration file against the database.
+2. It runs the SQL migration file against the database.
+3. Installs Prisma Client and generates a client based on yur models
```

Comment:
This is unfortunately not fully correct. 
`prisma migrate dev` runs `prisma generate` under the hood, which installs Prisma Client and generates a Client.

Reply:
I've made the suggested change @janpio Kindly take a look